client/src/components/streams/StreamShow.js

```diff
@@ -49,22 +49,29 @@ class StreamShow extends React.Component {
       <div id="stream-show" className="ui grid">
         <div id="stream-container" className="twelve wide column">
           <div className="ui secondary menu header">
-            <div className="item">{username || "User"}'s Stream</div>
+            <div className="item" style={{ color: "white" }}>
+              {username || "User"}'s Stream
+            </div>
             <div
               className="item right"
-              style={{ paddingTop: 0, paddingBottom: 0 }}
+              style={{ paddingTop: 10, paddingBottom: 0 }}
             >
-              <button className="ui red button floated">Subscribe</button>
+              <button className="ui teal button floated">Subscribe</button>
             </div>
           </div>
-          <div className="content-scrollable">
+          <div
+            className="content-scrollable"
+            style={{ color: "white", paddingLeft: 5 }}
+          >
             <video ref={this.videoRef} style={{ width: "100%" }} controls />
-            <h1>{title}</h1>
-            <h4>#{gametag}</h4>
-            <h5>{description}</h5>
+            <div style={{ padding: 10 }}>
```

Comment:
I think this is over-riding the default padding from semantic and making stuff look uncentered. Either that or line something with line 57. The smaller gap does look better, but the username being uncentered isn't ideal.
Before:
![image](https://user-images.githubusercontent.com/25234283/79010890-1a61ac00-7b20-11ea-9f01-0bca55919660.png)

After:
![image](https://user-images.githubusercontent.com/25234283/79010838-00c06480-7b20-11ea-81d8-6084dc355335.png)